### ISSUE TITLE
Timezone & Coordinates with Nationality Makes No Sense

### ISSUE THREAD

**Issue by Modelmat** [RandomAPI/Randomuser.me-Node#119]
As seen here https://randomuser.me/api/?format=pretty&nat=au&inc=location, both the coordinates and the time zone fields often fall way outside the areas in which the address falls, for example:
```
{
  "results": [
    {
      "location": {
        "street": "4690 washington ave",
        "city": "bowral",
        "state": "western australia",
        "postcode": 3058,
        "coordinates": {
          "latitude": "-36.5799",
          "longitude": "89.7345"
        },
        "timezone": {
          "offset": "+5:00",
          "description": "Ekaterinburg, Islamabad, Karachi, Tashkent"
        }
      }
    }
  ],
}
```
Makes no sense as the timezones in Australia do not fall here, and the coordinates are in the middle of the ocean! (https://www.latlong.net/c/?lat=-39&long=89).

Please fix this.

**Comment by bbezanson** [comment]
> Makes no sense as the timezones in Australia do not fall here, and the coordinates are in the middle of the ocean! (https://www.latlong.net/c/?lat=-39&long=89).
>
> Please fix this.

I agree, I noticed this right away. Both should match the address -- but there are public APIs to get the timezone for an address or lat/long and get lat/long from an address as well.

**Comment by JackStratford** [comment]
Latitude and Longitude don't match the country and city as well as timezone i random so in the end you get data from 3 different locations. 

location:
city: "Spijk gem Lingewaal"
coordinates: {latitude: '38.3252', longitude: '111.4167'} <<Somewhere in China>>
country: "Netherlands"
postcode: 88471
state: "Overijssel"
street: {number: 8978, name: 'Dannenberg'}
timezone: {offset: '+11:00', description: 'Magadan, Solomon Islands, New Caledonia'}
